src/main/java/com/ernie/TicketApp/controller/UserController.java

```diff
@@ -0,0 +1,41 @@
+package com.ernie.TicketApp.controller;
+
+import com.ernie.TicketApp.model.User;
+import com.ernie.TicketApp.service.TicketService;
+import com.ernie.TicketApp.service.UserService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+
+    private final TicketService ticketService;
+    private final UserService userService;
+
+    public UserController(TicketService ticketService, UserService userService) {
+        this.ticketService = ticketService;
+        this.userService = userService;
+    }
```

Comment:
instead of that you can use `@RequiredArgsConstructor` or `@AllArgsConstructor` from Lombok lib